theme_5/src/styles/vendor/categorie-product.scss

```diff
@@ -0,0 +1,75 @@
+
+// CATEGORIE PRODUCT STYLES
+
+.products-container {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 100px;
+  padding: 0 15px;
+}
+
+.products-list {
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  margin-bottom: 20px;
+  width: 100%;
+}
+
+.product-link {
+	color: $black-solid;
+  top: 25px;
+
+	&:hover {
+		color: $black-solid;
+	}
+
+	&:active {
+		color: $black-solid;
+	}
+}
+
+.product-item {
+	border: 1px solid $cerise-red;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered align-items, border, display, flex-direction, justify-content, margin, min-height, text-align, width